src/util/helpers.js

```diff
@@ -48,9 +48,10 @@ const getVersionFromConfig = (resourceString) => {
  */
 const parseResourceVersions = (resourceString) => {
   if (!resourceString) return {}
-  const resourceFormatRegex = /(([A-Za-z])\w*)=([0-9]+).([0-9]+)([^;:|],*)/g
+  const resourceFormatRegex = /((([A-Za-z])+)=([0-9]+)([. 0-9]*)(,?))+/
+  const match = resourceString.match(resourceFormatRegex)
   const noSpResources = resourceString.replace(/\s/g, '')
-  if (!resourceFormatRegex.test(noSpResources)) {
+  if (!(match && resourceString === match[0])) {
```

Comment:
Isn't "test()" slightly more effecient?

Reply:
Its not in that case.
it gives *true* if the part of string matches, but not the whole part...